--- node_modules/musicmetadata/lib/browser.js	2015-09-14 18:17:10.000000000 +0200
+++ node_modules/musicmetadata_changed/lib/browser.js	2016-04-18 14:24:28.546604361 +0200
@@ -6,10 +6,10 @@
 var Stream = require('stream').Stream
 
 module.exports = function (stream, opts, callback) {
-  return musicmetadata(wrapFileWithStream(stream), opts, callback)
+  return musicmetadata(wrapFileWithStream(stream, opts), opts, callback)
 }
 
-function wrapFileWithStream (file) {
+function wrapFileWithStream (file, opts) {
   var stream = through(function write (data) {
     if (data.length > 0) this.queue(data)
   }, null, {autoDestroy: false})
@@ -20,7 +20,7 @@
     })
   }
 
-  if (file instanceof Stream) {
+  if (file instanceof Stream || opts.assumeStream) {
     return file.pipe(stream)
   }
   if (file instanceof window.FileList) {
